qkeras/utils.py

```diff
@@ -430,3 +430,58 @@ def quantized_model_from_json(json_string, custom_objects=None):
   qmodel = model_from_json(json_string, custom_objects=custom_objects)
 
   return qmodel
+
+def load_qmodel(filepath, custom_objects=None, compile = True):
+  """
+  Load quantized model from Keras's model.save() h5 file.
+
+  # Arguments:
+        filepath: one of the following:
+                  - string, path to the saved model
+                  - h5py.File or h5py.Group object from which to load the model
+                  - any file-like object implementing the method `read` that returns
+                  `bytes` data (e.g. `io.BytesIO`) that represents a valid h5py file image.
+        custom_objects: Optional dictionary mapping names
+                  (strings) to custom classes or functions to be
+                  considered during deserialization.
+        compile: Boolean, whether to compile the model
+                  after loading.
+  
+  # Returns
+        A Keras model instance. If an optimizer was found
+        as part of the saved model, the model is already
+        compiled. Otherwise, the model is uncompiled and
+        a warning will be displayed. When `compile` is set
+        to False, the compilation is omitted without any
+        warning.
+  """
+
+  if not custom_objects:
+    custom_objects = {}
+
+  # let's make a deep copy to make sure our objects are not shared elsewhere
+  custom_objects = copy.deepcopy(custom_objects)
```

Comment:
Looks like duplicate code from [line408](https://github.com/google/qkeras/pull/17/files#diff-a88a3b892954722ae7315f170489bc2dR408) to [line429](https://github.com/google/qkeras/pull/17/files#diff-a88a3b892954722ae7315f170489bc2dR429)

Could you refactor this two function, so we only need to change custom_objects in one place/function?

Reply:
Thanks for reviewing! I refactored the code by adding another function to automatically add supported quantized layers into custom_objects.